pkg/obfuscator/noop_obfuscator.go

```diff
@@ -0,0 +1,26 @@
+package obfuscator
+
+// NoopObfuscator is only used for testing purposes across packages. It does obfuscate nothing.
```

Comment:
Then it should be in a test file.

Reply:
I thought it would work, but if it's a test file it can't be exported, this is needed in both cleaner and reporting packages.

Reply:
In that case copy it. Having test code in non-test files seems wrong.